src/components/Articles/articleComponents.test.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import Enzyme, { shallow } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+
+import { ArticleContainer } from './ArticlesContainer';
+import SingleArticleComponent from './SingleArticleComponent';
+import AllArticlesComponent from './AllArticlesComponent';
+
+
+Enzyme.configure({ adapter: new Adapter() });
+
+describe('<AllItemsComponent />', () => {
+  it('should match the snapshot', () => {
+    const articles = (<AllArticlesComponent debug />);
+    expect(articles).toMatchSnapshot();
+  });
+});
+
+describe('<ArticleContainer />', () => {
+  it('should match the snapshot', () => {
+    const articles = (<ArticleContainer debug />);
+    expect(articles).toMatchSnapshot();
+  });
+});
+
+describe('<SingleArticleComponent />', () => {
+  it('should match the snapshot', () => {
+    const articles = (<SingleArticleComponent debug />);
+    expect(articles).toMatchSnapshot();
+  });
+});
+
+describe('All articles', () => {
+  const props = {
+    articles: jest.fn(),
+  };
+
+  it('Renders articles', () => {
+    const articles1 = [{}];
+    const wrapper = shallow(<AllArticlesComponent articles={articles1} />);
+    expect(wrapper.find('.ui divided items')).toBeDefined();
+  });
+
+  it('Articles container', () => {
+    const wrapper = shallow(<ArticleContainer {...props} />);
+    expect(wrapper.find('.pusher-height')).toBeDefined();
+  });
+
+  it('Single Article component', () => {
+    const article = {
```

Comment:
This could be declared globally inside the props object, then all you have to do is pass props to the component.